contrib/generate-emulation.py

```diff
@@ -21,7 +22,46 @@
 from gi.repository import Json  # pylint: disable=wrong-import-position
 
 
-def _minimize_json(json_str: str) -> str:
+fu_chassis_kind = {
+    "FU_SMBIOS_CHASSIS_KIND_OTHER": 0x01,
```

Comment:
Maybe this enum should be exported from `libfwupd` instead.

Reply:
Okay, I'll import the shared library to replace this.